modules/mlflow/applications.tf

```diff
@@ -1,6 +1,6 @@
 module "minio" {
   app_name   = "mlflow-minio"
-  source     = "git::https://github.com/canonical/minio-operator//terraform?ref=track/ckf-1.9"
+  source     = "git::https://github.com/canonical/minio-operator//terraform?ref=kf-7014-open-tf-reqs"
```

Comment:
Should we wait for https://github.com/canonical/minio-operator/pull/204 to be merged instead of pinning to a dev branch?